web/packages/teleport/src/Account/Account.tsx

```diff
@@ -388,7 +388,7 @@ function PasskeysHeader({
       }
       description="Enable secure passwordless sign-in using
                 fingerprint or facial recognition, a one-time code, or
-                a device password."
+                a device password. Passkeys are usable for required multi-factor authentication checks for sessions."
```

Comment:
Suggestion: keep to indent, drop the "required" word (implied), be more general.

```suggestion
                a device password. Passkeys are usable for
                multi-factor authentication checks."
```